clientupdate/clientupdate_windows.go

```diff
@@ -55,3 +58,28 @@ func launchTailscaleAsGUIUser(exePath string) error {
 	}
 	return cmd.Start()
 }
+
+func wtsGetActiveSessionID() (uint32, error) {
+	var (
+		sessionInfo *windows.WTS_SESSION_INFO
+		count       uint32 = 0
+	)
+
+	const WTS_CURRENT_SERVER_HANDLE = 0
+	if err := windows.WTSEnumerateSessions(0, 0, 1, &sessionInfo, &count); err != nil {
+		return 0, fmt.Errorf("WTSEnumerateSessions: %w", err)
+	}
+	defer windows.WTSFreeMemory(uintptr(unsafe.Pointer(sessionInfo)))
```

Comment:
weird API taking a uintptr rather than something typed. and if it's untyped, why not a generic windows-wide Free thing, rather than WTSFree? anyway, no clue.

I did check that the windows.WTSEnumerateSession wrapper isn't doing any of its Go-level allocations, in which case this uintptr conversion would be hiding it from the GC.

It's all in C land (https://cs.opensource.google/go/x/sys/+/refs/tags/v0.14.0:windows/zsyscall_windows.go;l=4371-4377) so looks fine GC-wise.